### DISCUSSION TITLE
米家电热垫AB档位建议

### DISCUSSION THREAD

**Discussion by Oranjekop** [XiaoMi/ha_xiaomi_home#322] [Ideas]
现在是要自己输入AB区档位的方式，而且中间是-也没有限制，不是特别方便，能否更改为独立控制AB档位

**Comment by Oranjekop** [comment]
![image](https://github.com/user-attachments/assets/02cc8959-0f2c-40e4-9744-4902e161b84c)

**Comment by SusanPhevos** [comment]
xiaomi_home 按照厂商定义的 MIoT-Spec-V2 生成实体，给设备发送的控制命令也要符合定义的 MIoT-Spec-V2 格式。
详见：https://github.com/XiaoMi/ha_xiaomi_home/blob/main/doc/README_zh.md#%E4%B8%80%E8%88%AC%E8%BD%AC%E6%8D%A2%E8%A7%84%E5%88%99
xiaomi_home 在 UI 上不会再做变更。
